src/containers/blogs/Blogs.js

```diff
@@ -1,11 +1,47 @@
-import React, {useContext} from "react";
+import React, {useState, useEffect, useContext} from "react";
 import "./Blog.scss";
 import BlogCard from "../../components/blogCard/BlogCard";
 import {blogSection} from "../../portfolio";
 import {Fade} from "react-reveal";
 import StyleContext from "../../contexts/StyleContext";
 export default function Blogs() {
   const {isDark} = useContext(StyleContext);
+  const [mediumBlogs, setMediumBlogs] = useState([]);
+  function setMediumBlogsFunction(array) {
+    setMediumBlogs(array);
+  }
+  function extractContent(html) {
+    return typeof html === "string"
+      ? html
+          .split("p>")
+          .filter(el => !el.includes(">"))
+          .map(el => el.replace("</", ".").replace("<", ""))
+          .join(" ")
+      : NaN;
+  }
```

Comment:
Can you add a comment on what this function does?

Reply:
done: done: https://github.com/saadpasta/developerFolio/pull/445/commits/176cfb186e8c74ec68002c11ee651f0894e6174b